src/index.js

```diff
@@ -49,7 +49,13 @@ class HtmlWebpackJsdomPrerenderPlugin {
     const source = asset.source()
 
     const baseUrl = 'http://localhost'
-    const scriptUrl = new URL(outputFile, baseUrl).href
+    let filePath = outputFile
+    const publicPath = compilation.options.output?.publicPath
```

Comment:
Non-blocking: Is this option documented anywhere? I saw an example in [the README](https://github.com/associatedpress/html-webpack-jsdom-prerender-plugin/blob/main/README.md#getting-started), but that doesn't need to be in this PR

Reply:
It's just being pulled from the webpack config, so it's not an option you have to pass directly to the plugin. But I can def add a note to the readme making it clear what types of publicPath are supported.